app/assets/javascripts/components/Glossary.js

```diff
@@ -0,0 +1,90 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Glossary;
+
+  Glossary = function($el, config) {
+    Glossary.baseConstructor.call(this, $el, config);
+
+    var article = $('[data-dough-glossary-content]'),
+        articleContent = $(article).html(),
+        updatedContent;
+
+    $.getJSON('/glossary.json', function(data){
+      data.glossary.map(function (item) {
+
+    var keywordInArticleContent = function(article, term) {
+      return articleContent.indexOf(item.term) >= 1;
```

Comment:
This should be `return article.indexOf(term) >= 1;`. Although I would be inclined to refactor to

```
function keywordInContent(keyword, content){
  return content.indexOf(keyword)>= 1;
}```

and then line 22 would be `if(keywordInContent(item.term, articleContent)`